packages/social-db-api/src/types.ts

```diff
@@ -31,6 +37,7 @@ export type SocialGetParams = (
   | {
       key?: never;
       keys: string[];
+      options?: SocialGetOptions;
```

Comment:
Would it make sense to adjust the type to always allow `options` (regardless if you're passing `key` or `keys`)?

```ts
export type SocialGetParams = {
  options?: SocialGetOptions;
} & (
  | {
      key: string;
      keys?: never;
    }
  | {
      key?: never;
      keys: string[];
    }
) &
  (
    | {
        blockId?: never;
        finality?: Finality;
      }
    | {
        blockId: BlockId;
        finality?: never;
      }
  );
```

Reply:
Yeah, I totally agree with this one. Technically options are not dependent on the payload structure, it's just a param that we could configure for the request. @calebjacob thanks for the code snippet!